.changeset/pink-guests-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-tooltip": minor
+---
+
+Added an optional style prop to Tooltip
```

Comment:
I think you can update this description now that you're not adding a whole `style` prop anymore.